src/wui/http/http_client.c

```diff
@@ -315,6 +312,9 @@ http_wait_headers(struct pbuf *p, u32_t *content_length, u16_t *total_header_len
                         header_info.content_type = TYPE_JSON;
                     } else if (0 == strncmp(content_type_str, type_xgcode_str, strlen(type_xgcode_str))) {
                         header_info.content_type = TYPE_GCODE;
+                    } else {
+                        header_info.valid_request = false;
+                        return ERR_VAL; // quit the connection if content type is un-supported
```

Comment:
In http_client.h we have this as an error:
`{ "error with Content-Type", CMD_REJT_CONT_TYPE }`
I think it's reasonable to return REJECT event saying there is Content-Type wrong.. If you return ERR_VAL, it won't respond, it'll just ignore the request.

Reply:
it is not problem to remove that error type. because Ondra's server will send rubbish content types, and he said it is not necessary to respond to all what is received